packages/core/content-manager/admin/src/pages/EditView/EditViewPage.tsx

```diff
@@ -180,7 +168,14 @@ const EditViewPage = () => {
         disabled={hasDraftAndPublished && status === 'published'}
         initialValues={initialValues}
         method={isCreatingDocument ? 'POST' : 'PUT'}
-        validate={validate}
+        validate={(values: Record<string, unknown>, options: Record<string, string>) => {
+          const yupSchema = createYupSchema(schema?.attributes, components, {
+            status,
+            ...options,
+          });
+
+          return yupSchema.validate(values, { abortEarly: false });
+        }}
```

Comment:
Sorry, previously I was asking if it is possible to use the validate function useDocument exposes:

https://github.com/strapi/strapi/blob/46c3b79e02bbbbb309f7420bcace00a271149d2e/packages/core/content-manager/admin/src/hooks/useDocument.ts#L127-L168

If not, do you know what the difference between the too is?

Reply:
I think here we should use the validate function coming from the Form context, because it's the one that is tied to the form inputs, in order to display validation errors within the fields.

So I think the change would need to be done higher up, because it comes from something like that:

```js
<Form validation={someValidateFn} />
```

and I agree with you that it sounds like we could use the useDocument fn there. But it's also not related to cmd+enter to save, so I don't want to do it in this PR. It sounds like it could be part of a larger topic to avoid rerenders and reduce tech debt

Reply:
👍  That makes sense. Thanks.